dsc/src/util.rs

```diff
@@ -268,46 +287,78 @@ pub fn write_output(json: &str, format: &Option<OutputFormat>) {
     }
 }
 
-pub fn enable_tracing(trace_level: &Option<TraceLevel>, trace_format: &TraceFormat) {
-    let tracing_level = match trace_level {
-        Some(level) => level,
-        None => {
-            // use DSC_TRACE_LEVEL env var if set
-            match env::var(DSC_TRACE_LEVEL) {
-                Ok(level) => {
-                    match level.to_ascii_uppercase().as_str() {
-                        "ERROR" => &TraceLevel::Error,
-                        "WARN" => &TraceLevel::Warn,
-                        "INFO" => &TraceLevel::Info,
-                        "DEBUG" => &TraceLevel::Debug,
-                        "TRACE" => &TraceLevel::Trace,
-                        _ => {
-                            warn!("Invalid DSC_TRACE_LEVEL value '{level}', defaulting to 'warn'");
-                            &TraceLevel::Warn
-                        },
-                    }
+pub fn enable_tracing(trace_level_arg: Option<TraceLevel>, trace_format_arg: Option<TraceFormat>) {
+    
+    let mut policy_is_used = false;
+    let mut tracing_setting = TracingSetting::default();
+
+    // read setting/policy from files
+    if let Ok(v) = get_setting("tracing") {
+        if v.policy != serde_json::Value::Null {
+            match serde_json::from_value::<TracingSetting>(v.policy) {
+                Ok(v) => {
+                    tracing_setting = v;
+                    policy_is_used = true;
+                },
+                Err(e) => { println!("{e}"); }
```

Comment:
If there's an error, we should just go back to default and use `error!`

Reply:
tracing infrastructure is Not initialized at this point, so errors are just silently swallowed.
So I thought that printing to console in the next best thing.

Reply:
In this case, we can still init to default and use tracing.

Reply:
Updated.